lib/export-helpers/user-helper.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const MsCall = require('@janiscommerce/microservice-call');
+const ExportHelper = require('./export-helper');
+
+class ExportUserHelper extends ExportHelper {
+
+	/**
+	 *
+	 * @static
+	 * @param {Array} items
+	 * @param {Object} session
+	 * @returns
+	 * @memberof ExportUserHelper
+	 */
+	static async getUsers(items, session) {
+		this.items = items;
+
+		const userIds = this.getIds();
+
+		const microServiceCall = new MsCall(session);
```

Comment:
Hay algo extraño con esta linea porque no veo en el paquete de microservice-call que pueda instanciarse de esta manera. No hay un constructor que reciba session. En tal caso, estarías usando una librería propia que wrappee esto. 
![image](https://user-images.githubusercontent.com/29028845/81426606-d1f1da00-912f-11ea-9f89-76a01fa80741.png)

Reply:
Estas en lo correcto no va a funcionar asi, para sessionar el MsCall lo corrector seria hacer algo asi:

```suggestion
		const microServiceCall = session.getSessionInstance(MsCall);
```